roblox/partials/partialuser.py

```diff
@@ -79,3 +79,25 @@ def __init__(self, client: Client, data: dict):
         super().__init__(client=client, data=data)
 
         self.previous_usernames: List[str] = data["previousUsernames"]
+
+
+class CatalogCreatorPartialUser(PartialUser):
```

Comment:
I don't think a new class is warranted here. I think, unless it has extra fields, all PartialUser-shaped objects should just be PartialUsers. Maybe PartialUser could optionally take fields directly , like `PartialUser(client=..., id=..., name=..., has_verified_badge=...)` so we don't have to make more classes.